addons/account_update_tax_tags/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="action_open_wizard" model="ir.actions.act_window">
+            <field name="name">Update tax tags on existing Journal Entries</field>
+            <field name="res_model">account.update.tax.tags.wizard</field>
+            <field name="view_mode">form</field>
+            <field name="target">new</field>
+        </record>
+
+        <record id="res_config_settings_view_form" model="ir.ui.view">
+            <field name="name">res.config.settings.view.form.inherit.account_update_tax_tags</field>
+            <field name="model">res.config.settings</field>
+            <field name="inherit_id" ref="account.res_config_settings_view_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//div[@name='default_taxes_setting_container']" position="inside">
+                    <button name="%(account_update_tax_tags.action_open_wizard)d" icon="fa-refresh" type="action" class="btn-link"
+                            groups="account.group_account_manager"
```

Comment:
For now it's not auto installed, although we can discuss whether it should be.
Benj told me he changed his mind regarding the debug mode, he thinks it should be visible all the time for authorized users, since those are already limited. And since we have a big warning in the wizard it should be safe enough.